src/ontology/mondo-edit.obo

```diff
@@ -347005,20 +346944,19 @@ is_a: MONDO:0019512 {source="Orphanet:363189"} ! congenital heart malformation
 
 [Term]
 id: MONDO:0018186
-name: ring chromosome
-def: "Aberrant chromosomes with no ends, i.e., circular." [MESH:D012303]
+name: obsolete ring chromosome
+def: "OBSOLETE. Aberrant chromosomes with no ends, i.e., circular." [MESH:D012303]
 comment: Reason: grouping class. This term is from orphanet and refers specifically to autosomal chromosome. Consider MONDO:0700091, refering to both autosomal and gonosomal chromosomes. Term to consider: MONDO:0700091
 subset: disease_grouping
-subset: obsoletion_candidate
 subset: ordo_group_of_disorders {source="Orphanet:363203"}
 synonym: "supernumerary circular chromosome" BROAD [NCIT:C3360]
 xref: ICD10:Q93.2 {source="ORDO:363203/attributed", source="ORDO:363203/ntbt", source="Orphanet:363203"}
 xref: MESH:D012303 {source="MONDO:subClassOf"}
 xref: NCIT:C3360 {source="MONDO:subClassOf"}
-xref: Orphanet:363203 {source="MONDO:equivalentTo"}
-is_a: MONDO:0015153 {source="Orphanet:363203"} ! autosomal monosomy
+xref: Orphanet:363203 {source="MONDO:obsoleteEquivalent"}
 relationship: seeAlso https://github.com/monarch-initiative/mondo/issues/4272
-property_value: IAO:0006012 "2022-04-01" xsd:string
+property_value: IAO:0000231 "grouping class" xsd:string
```

Comment:
we are ultimately planning to standardize the obsoletion reasons, and for grouping classes, I've been using the reason 'out of scope' for grouping classes.

Could you please update the obsoletion reasons to be 'out of scope' instead of 'grouping class'.

I think you can just go into the text file and do a search and replace (search for "grouping class" xsd:string and replace with "out of scope" xsd:string)

Reply:
see: https://github.com/monarch-initiative/mondo/discussions/2765

I thought there was a ticket for to change all the obsoletion reasons to individuals, but I can't find it

Reply:
Note to self: based on the editor note for "out of scope" in [this document](https://github.com/information-artifact-ontology/ontology-metadata/blob/master/src/ontology/omo-edit.owl): "Typical valid examples are: un-necessary grouping classes in disease ontologies"
So that is the reason to use "out of scope" in this case.
Is that correct?

Reply:
yes, that is my understanding. Let's just use 'out of scope' for now to be consistent but at some point we can decide if we need other obsoletion reasons. @sabrinatoro can you add any questions to the discussion (https://github.com/monarch-initiative/mondo/discussions/2765) about obsoletions and make sure this is on a future agenda doc?